src/app/ApplicationTemplate.Presentation/Framework/Review/ReviewSettingsSource.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ApplicationTemplate;
+
+/// <summary>
+/// Implementation of <see cref="IReviewSettingsSource{TReviewSettings}"/>.
+/// </summary>
+public sealed class ReviewSettingsSource : IReviewSettingsSource<ReviewSettings>
```

Comment:
I don't think this makes sense. What is this implementation supposed to be relying on?